mm/ktsan/thr.c

```diff
@@ -38,9 +39,13 @@ kt_thr_t *kt_thr_create(kt_thr_t *thr, int kid)
 		pr_err("KTSAN: maximum number of threads is reached\n");
 		BUG();
 	}
+	/* Kernel does not assign PIDs to some threads, give them fake
+	 * negative PIDs so that they are distinguishable in reports. */
+	if (pid == 0)
+		pid = --pool->new_pid;
```

Comment:
I suspect that there is no actual thread with PID 0, because PID 0 usually means something special in syscalls. For example, man kill:
>       If pid equals 0, then sig is sent to every process in the process group of the calling process.